db/migrate/20150107044854_users_without_location_cleanup.rb

```diff
@@ -0,0 +1,9 @@
+class UsersWithoutLocationCleanup < ActiveRecord::Migration
+  def up
+    User.delete_all(location: [nil,''])
```

Comment:
This error has happened **42** times. 

**First** seen on Nov 29 2014, 09:44:56 AM. 
**Last** seen on Jan 05 2015, 04:48:25 AM.